cucoslib/workers/graphaggregator.py

```diff
@@ -21,8 +25,25 @@ def _handle_external_deps(self, ecosystem, deps):
         versions = solver.solve(deps)
         return [{"package": k, "version": v} for k, v in versions.items()]
 
+    def store_in_bucket(self, content):
+        bucket_name = 'janus-json-data'
+        try:
+            email = content.get('email')
+            s3 = AmazonS3(bucket_name=bucket_name)
+            s3.connect()
+
+            folder = 'new_user_profile'
+            key = folder + '/' + email + '.json'
+            s3.store_dict(content, key)
+        except:
+            raise Exception('Amazon Exception')
```

Comment:
You probably don't want to do this as these errors are tracked inside aggregated logs. Re-raising here does not add any additional info for exception.